### PR TITLE
Homepage/hero

### PR DESCRIPTION
**Pull request by tammalee**
Styling for Hero! 
Screenshot 2019-12-13 10 57 07

Screenshot 2019-12-13 10 56 30

Screenshot 2019-12-13 10 56 25

Screenshot 2019-12-13 10 19 46

Note: The look ahead functionality will be styled in the next PR where I adjust all existing `$brand-color` variables. It currently looks like this and that's A-ok.
Screenshot 2019-12-13 10 20 13

### CODE REVIEW

File: styles/_base.scss [hunk 1]
```diff
@@ -18,35 +18,80 @@ body {
   }
 
   > main {
```
Comment (eddysims) [inline styles/_base.scss]: Can we not style tags directly.

File: styles/_header.scss [hunk 2]
```diff
@@ -17,9 +17,9 @@
 .logo {
 
   &__link {
-    
-    &:hover {
-      text-decoration: none;
+    box-shadow: unset;
+    &:hover  {
```
Comment (eddysims) [inline styles/_header.scss]: extra space here

File: styles/_header.scss [hunk 3]
```diff
@@ -17,9 +17,9 @@
 .logo {
 
   &__link {
-    
-    &:hover {
-      text-decoration: none;
+    box-shadow: unset;
```
Comment (eddysims) [inline styles/_header.scss]: missing blank line

File: styles/_hero.scss [hunk 4]
```diff
@@ -1,16 +1,51 @@
...
 .hero {
-  background-image: url($homepage_background_image);
-  background-position: center;
-  background-size: cover;
-  height: 300px;
-  padding: 0 20px;
+  background-color: $secondary;
+  @include fluid-value( height, $min-width, $max-width, 300px, 550px);
+  @include fluid-value( padding-left, $min-width, $max-width, $spacing, $spacing--desktop);
+  @include fluid-value( padding-right, $min-width, $max-width, $spacing, $spacing--desktop);
```
Comment (eddysims) [inline styles/_hero.scss]: there are spaces the first bracket but not the last

File: styles/_hero.scss [hunk 5]
```diff
@@ -1,16 +1,51 @@
...
   text-align: center;
   width: 100%;
 
   &-inner {
     @include vertical-align;
-    max-width: 610px;
+    max-width: 750px;
```
Comment (eddysims) [inline styles/_hero.scss]: should this be a variable?

File: styles/_hero.scss [hunk 6]
```diff
@@ -1,16 +1,51 @@
...
     margin: 0 auto;
+    
+    h1 {
```
Comment (eddysims) [inline styles/_hero.scss]: Can we change this to a class?

File: templates/home_page.hbs [hunk 7]
```diff
@@ -1,72 +1,21 @@
...
-                </a>
-              </li>
-            {{/each}}
-          {{/if}}
+  <div class="container hero-inner">
+    <h1>How can we help?</h1>
+    {{search submit=false placeholder='Search Help Center' instant=settings.instant_search class='search search-full'}}
+    {{#if promoted_articles}}
+      <ul class="hero__popular-topics" role="list">
+        <li class="hero__popular-topics--item" role="listitem">Popular Topics:</li>
```
Comment (eddysims) [inline templates/home_page.hbs]: for bem `--` is a modifier. To follow proper bem, here we would something like `hero__topic-item`

File: templates/home_page.hbs [hunk 8]
```diff
@@ -1,72 +1,21 @@
...
+        {{#each promoted_articles}}
+          <li class="hero__popular-topics--item" role="listitem">
+            <a class="hero__popular-topics--link" href="{{url}}">
```
Comment (eddysims) [inline templates/home_page.hbs]: same bem comment as above

File: templates/home_page.hbs [hunk 9]
```diff
@@ -1,72 +1,21 @@
...
-                    {{name}}
-                  </span>
-                  <span class="blocks-item-description">{{excerpt description}}</span>
-                </a>
-              </li>
-            {{/each}}
-          {{/if}}
+  <div class="container hero-inner">
+    <h1>How can we help?</h1>
+    {{search submit=false placeholder='Search Help Center' instant=settings.instant_search class='search search-full'}}
```
Comment (ericfranck) [inline templates/home_page.hbs]: Can we change this to "Search the Help Center"?

**Comment by eddysims** [comment]
I feel that the border transition is sharp, can we slow that down a bit?

File: styles/_base.scss (hunk 1 above)
Comment (tammalee) [reply styles/_base.scss]: A lot of this is the ZenDesk code and not something I've written. 
I won't refactor it all but I'll see what I can do.

File: styles/_hero.scss (hunk 5 above)
Comment (tammalee) [reply styles/_hero.scss]: It's a one-off value. Should it be?

File: templates/home_page.hbs (hunk 9 above)
Comment (tammalee) [reply templates/home_page.hbs]: Totes McGotes!

**Pull request merged**